core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -75,9 +75,11 @@ class KryoSerializerSuite extends SparkFunSuite with SharedSparkContext {
   }
 
   test("basic types") {
-    val ser = new KryoSerializer(conf).newInstance()
     def check[T: ClassTag](t: T) {
-      assert(ser.deserialize[T](ser.serialize(t)) === t)
+      testBothUnsafeAndSafe(conf) { sparkConf =>
```

Comment:
(We have a few other examples suites where we just inherit from existing suites, such as SortShuffleSuite vs ShuffleSuite.)